ci/docker/.env

```diff
@@ -0,0 +1,4 @@
+IMAGE_NAME=emcmongoose/mongoose-base
+IMAGE_TAG=latest
+COMPOSE_PROJECT_NAME=mongoose
+
```

Comment:
I don't see those variables are used anywhere within the Pull Request. The .env file itself (as I can see) isn't passed via docker-related files as well. Am I missing something?

Reply:
This file is applied automatically.